easybuild/easyconfigs/c/Caffe/Caffe-1.0-intel-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,69 @@
+# -*- mode: python; -*-
+# EasyBuild reciPY for Caffe as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2017 UL HPC
+# Authors::   UL HPC Team <hpc-sysadmins@uni.lu>
+# License::   MIT/GPL
+#
+easyblock = 'CMakeMake'
+
+name = 'Caffe'
+version = '1.0'
+
+# cudaversion = '7.5.18'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/BVLC/caffe'
+description = """
+Caffe is a deep learning framework made with expression, speed,
+and modularity in mind. It is developed by the Berkeley Vision
+and Learning Center (BVLC) and community contributors.
+"""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+source_urls = ['https://github.com/BVLC/caffe/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['71d3c9eb8a183150f965a465824d01fe82826c22505f7aa314f700ace03fa77f']
+
+
+builddependencies = [
+    ('CMake', '3.8.1'),
```

Comment:
try to use the same CMake version consistently, latest available compatible with `intel/2017a` is 3.9.1